vision_msgs_rviz_plugins/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vision_msgs_rviz_plugins</name>
+  <version>0.1.0</version>
+  <description>RVIZ2 plugins for visualizing vision_msgs</description>
+  <maintainer email="georg.novotny@aon.at">Georg Novotny</maintainer>
+  <maintainer email="stevenmacenski@gmail.com">Steve Macenski</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
+
+
+  <depend>yaml-cpp</depend>
+  <depend>rviz2</depend>
+
+
+  <depend>rclcpp</depend>
+  <depend>rclpy</depend>
+  <depend>pluginlib</depend>
+
+  <depend>rviz_common</depend>
+  <depend>rviz_default_plugins</depend>
+  <depend>rviz_rendering</depend>
+  <depend>vision_msgs</depend>
+  
+  <depend>tf_transformations</depend>
```

Comment:
I don't think this is a rosdep key

Reply:
Don't worry about the multiple rounds. I am sorry that it is needed.

The rosdep keys should be valid since:

1. I build everything in a docker (ros:humble-ros-core-jammy) and ran `rosdep install --from-paths src --ignore-src -r -y` without error.
2. During earlier commits the [build on the ros buildfarm failed](https://build.ros2.org/job/Hpr__vision_msgs__ubuntu_jammy_amd64/6/console#console-section-9) if it could not reslove the rosdep keys. Here it could not find _qt5-default_. On the latest commit the buildfarm did not throw any errors and could resolve the keys:
![Screenshot from 2023-01-27 20-17-04](https://user-images.githubusercontent.com/31406662/215177125-d40fbda3-b5d3-4434-babf-cc10642e0623.png)

Reply:
https://github.com/ros/rosdistro/blob/c95e19fa3c38ca124982f66db59f907fecc3d6aa/foxy/distribution.yaml#L2505 is preferred so that it can build on other non-linux OS's as well since ROS 2 is supposed to support WIndows and MacOS

Reply:
tf_transformations is definitely not right though

Reply:
The tf_transformations is basically a wrapper around the new transforms3d that provides seamless  backward compability. As far as I understand the tf_transformations is also a valid rosdep key since it is in this repo: https://github.com/ros/rosdistro/blob/master/humble/distribution.yaml#L6301
and can be installed via apt as: ros-humble-tf-transformations

Reply:
My latest push includes now the yaml_cpp_vendor: https://github.com/NovoG93/vision_msgs/blob/ros2/vision_msgs_rviz_plugins/package.xml#L15
Should be fine now?

Reply:
I guess I've very much prefer not introducing something like that into an interface visualization library since we don't know how long if / when David might stop releasing it to new distributions. TF we know will be around for as long as ROS is around and can have the same type of function to convert RPY to quaternions https://answers.ros.org/question/69754/quaternion-transformations-in-python/ (or even just that's a trivial function to implement either way and I'm sure there are batteries included python libraries that deal with that kind of thing)

Reply:
Okay, now I understand why you want me to switch.
Will do so👍